package.json

```diff
@@ -51,13 +51,17 @@
   "dependencies": {
     "@babel/polyfill": "^7.0.0",
     "autoprefixer": "^9.1.3",
+    "chalk": "^2.4.1",
```

Comment:
Shouldn't this be in the `devDependencies`, it's not meant to be used in production, or bundled in the final build, right?